gdx/src/com/badlogic/gdx/graphics/g3d/particles/influencers/DynamicsModifier.java

```diff
@@ -415,11 +421,17 @@ public void update () {
 
 				float cosTheta = MathUtils.cosDeg(theta), sinTheta = MathUtils.sinDeg(theta), cosPhi = MathUtils.cosDeg(phi), sinPhi = MathUtils
 					.sinDeg(phi);
-				TMP_V3
-					.set(cosTheta * sinPhi, cosPhi, sinTheta * sinPhi)
-					.crs(positionChannel.data[positionOffset + ParticleChannels.XOffset],
-						positionChannel.data[positionOffset + ParticleChannels.YOffset],
-						positionChannel.data[positionOffset + ParticleChannels.ZOffset]).nor().scl(strength);
+				TMP_V3.set(cosTheta * sinPhi, cosPhi, sinTheta * sinPhi);
+				TMP_V1.set(positionChannel.data[positionOffset + ParticleChannels.XOffset],
+					positionChannel.data[positionOffset + ParticleChannels.YOffset],
+					positionChannel.data[positionOffset + ParticleChannels.ZOffset]);
+				if (!isGlobal) {
```

Comment:
if isGlobal == false, then we need to get the position of the particle relative to the center rather than the origin, so we subtract the controller's position from TMP_V1.
TMP_V3 needs to be rotated if the controller was so that the axis about which we apply our force is relative to the controller rather than to the origin.